loki/ir/nodes.py

```diff
@@ -560,7 +570,7 @@ class _ConditionalBase():
 
     condition: Expression
     body: Tuple[Node, ...]
-    else_body: Optional[Tuple[Node, ...]] = None
+    else_body: Optional[Tuple[Node, ...]] = ()
```

Comment:
Is that a change in behaviour for the default value of `else_body`?
I think so far we used `else_body=None` as indicator that there isn't an else branch in the conditional - as opposed to an empty else branch (which has the same result but different on a string level).

As in, will `ir.Conditional(condition=<...>, body=<...>)` now be represented as the following?
```
IF (<...>) THEN
  <...>
ELSE
ENDIF
```